sc-network/sctp_client/CMakeLists.txt

```diff
@@ -21,11 +21,16 @@ else ()
         "sockets/glibSocket.cpp"
         "sockets/glibSocket.hpp"
     )
-    set (SOCKET_LIBS  ${GLIB2_LIBRARIES})
+    set (SOCKET_LIBS  ${GLIB2_INCLUDE_DIRS})
```

Comment:
Why we put include directories into libraries list?

Reply:
Returned `GLIB2_LIBRARIES` as it was